package.json

```diff
@@ -123,5 +123,9 @@
       "eslint --fix"
     ]
   },
+  "resolutions": {
```

Comment:
This shouldn't be needed? If you upgrade `react-error-boundary` too, then there is no conflict I think.

Reply:
Looks like upgrading that library prevents the resolution, but the API on `react-error-boundary` has changed in the newer version.

The /site/pages/examples/[example].tsx can be changed to something like this to get it working.

```
const ExamplePage = ({ example }: { example: string }) => {
  const [showTabs, setShowTabs] = useState<boolean>()
  const EXAMPLE = EXAMPLES.find(e => e[2] === example)
  const [name, Component, path] = EXAMPLE
  return (
    <ErrorBoundary
      fallbackRender={({ error }) => {
        return (<Warning>
          <p>An error was thrown by one of the example's React components!</p>
          <pre>
            <code>
              <>
                {error}
                {'\n'}
                {error.stack}
              </>
            </code>
          </pre>
        </Warning>)
      }}
    >
      <div>
        <Head>
          <title>Slate Examples</title>
          <meta name="viewport" content="width=device-width, initial-scale=1" />
          <link rel="icon" href="/favicon.ico" />
          <link rel="stylesheet" href="/index.css" />
          <link
            rel="stylesheet"
            href="https://fonts.googleapis.com/css?family=Roboto:400,400i,700,700i&subset=latin-ext"
          />
          <link
            rel="stylesheet"
            href="https://fonts.googleapis.com/icon?family=Material+Icons"
          />
        </Head>
        <Header>
          <Title>Slate Examples</Title>
          <LinkList>
            <A href="https://github.com/ianstormtaylor/slate">GitHub</A>
            <A href="https://docs.slatejs.org/">Docs</A>
          </LinkList>
        </Header>
        <ExampleHeader>
          <TabButton
            onClick={e => {
              e.stopPropagation()
              setShowTabs(!showTabs)
            }}
          >
            <Icon>menu</Icon>
          </TabButton>
          <ExampleTitle>
            {name}
            <A
              href={`https://github.com/ianstormtaylor/slate/blob/main/site/examples/${path}.tsx`}
            >
              (View Source)
            </A>
          </ExampleTitle>
        </ExampleHeader>
        <TabList isVisible={showTabs}>
          {EXAMPLES.map(([n, , p]) => (
            <Link
              key={p as string}
              href="/examples/[example]"
              as={`/examples/${p}`}
              passHref
            >
              <Tab onClick={() => setShowTabs(false)}>{n}</Tab>
            </Link>
          ))}
        </TabList>
        <ExampleContent>
          <Component />
        </ExampleContent>
        <TabListUnderlay
          isVisible={showTabs}
          onClick={() => setShowTabs(false)}
        />
      </div>
    </ErrorBoundary>
  )
}
```